portfolio/src/main/webapp/index.html

```diff
@@ -79,6 +79,8 @@ <h3>Overview</h3>
           </div>
           <div class="container">
             <form action="/comments" method="POST">
+              <textarea name="max-num"></textarea>
+              <br>
```

Comment:
Also as we discussed, this shouldn't be part of this form. Let's leave out the frontend change in this PR and only focus on the server changes, you can make another PR to connect it up.